src/main/java/edu/harvard/iq/dataverse/DatasetFieldServiceBean.java

```diff
@@ -246,6 +246,12 @@ public DatasetFieldType save(DatasetFieldType dsfType) {
         return em.merge(dsfType);
     }
 
+    /**
+     * Not sure why this method is here, better use the metadataBlockServiceBean
+     * @param mdb
+     * @return
+     */
+    @Deprecated
```

Comment:
You should add the `@deprecated` tag to the javadoc and provide a reason in the `@Deprecated` annotation, according to Sonarcloud.